### ISSUE TITLE
Introducing native support for *bounding asterisks*

### ISSUE THREAD

**Issue by geochronology** [adam-p/markdown-here#621]
I would like to propose the introduction of markdown support for \*bounding asterisks\*, with the goal of restoring a convention of internet communication and culture.

I believe that John Gruber's interpretation of bounding asterisks was, and remains, largely incorrect. To wit:
> By "emphasis" Gruber explains that he means "informing the reader of a shift in style or voice,"
> likening the use of bounding asterisks to "how foreign words are italicized in many publications
> and books." He figured it was an "Internet-ism," tracing its use to the need for a plain-text
> substitution for italicization or bolding.

That's not how people use bounding asterisks a large majority of the time. In internet culture (at least in internet subculture around gaming, anime, and so on), asterisks are not used to emphasize as often as they're used to \*emote\*.

The convention, for those unfamiliar, places a bounded phrase (written in third person) adjacent to speech, written in first person. For instance:

> \*Walks into the dark room, unable to see a thing\* Now where could I have put my cheese? 
> \*flicks on the light switch\* Ah, there it is! Right where I left it on the counter.

Another use of bounding asterisks is to indicate an onomonopea (`\*nom nom nom*\`) as referenced in the following [scholarly article](https://repository.library.northeastern.edu/downloads/neu:376918?datastream_id=content). To quote:

> Expressions can also be recreated by naming the expression offset by asterisks, for example 
> ‘*grins*’.  This technique can also be used to denote onomatopoeias like ‘*gluckgluckgluck*’
> recreating the sound of drinking quickly (Hentschel, 1998).

There are also instances in which, yes, bounding asterisks convey emphasis, but in my own experience this a much less frequent use case than the first one (though probably more frequent than the second). 

Since markdown has become quite influential, it's now impossible to write asterisk-bound phrases natively on Reddit, in addition to several prominent communication apps including Discord, Slack, and Skype. 

To me, this is an unintended consequence of the original spec that misreads a major communication convention of digital natives, and one that ought to be changed as to allow this format to return to its unformatted glory.

My alternative proposal for italics is bounded double exclamation, such as: "Can you !!believe!! we can't use bounded asterisks anymore? \*huffs\*"

I would very much like to know what others think about this, since it's an important issue deserving of a wider conversation.

**Comment by teh-maxh** [comment]
> "A plain-text substitution for italicization" is simply not why people use bounding asterisks online.

This is inaccurate. Asterisks as emphasis have a long history of use for emphasis. Your observation of other uses is correct, but to claim that it isn't used for emphasis is simply wrong.

**Comment by geochronology** [comment]
> 
> 
> > "A plain-text substitution for italicization" is simply not why people use bounding asterisks online.
> 
> This is inaccurate. Asterisks as emphasis have a long history of use for emphasis. Your observation of other uses is correct, but to claim that it isn't used for emphasis is simply wrong.

Yes, that's a fair point, and you're absolutely right, I've revised my argument accordingly:
> "A plain-text substitution for italicization" is just one of the reasons why people use bounding asterisks.
> In subcultures of anime, gaming, and fandom, for instance, bounding asterisks are used to *emote*,
> signifying that within the encapsulated phrase, an action is being taken.

**Comment by songcarver** [comment]
Small ideas: what if a period could be used to escape words or full lines. 
There is an existing use I've seen popularised in Twitter, where a period is used to 'escape' the @. 
For example, in twitter one writes
`@person123 check this out`, then it won't appear in the poster's main thread. Twitter users escape this by putting a period in front `.@person123 check this out'.

Let's say we wanted to write in a particular internet-y style.
`And *boom* she entered the room`

But escaping it, well, sucks, because it's so ugly

`And \*boom\* she entered the room`

Perhaps a period could do the trick for the word?

`And .*boom* she entered the room`

Or perhaps the full sentence can be escape

`.And *boom* she entered the room`
